### ISSUE TITLE
Problem in Recording's Playback after migrating to 2.5.5

### ISSUE THREAD

**Issue by alizamanir** [bigbluebutton/bigbluebutton#15655]
Hello
After I installed new 2.5.5 BBB Server and transferred the recording, the playback in Google Chrome faced with issue.
in Google Chrome its not possible to move the play back to forward or backward but it can be down in Firefox. 

I copied the Playback\2.3 folder from an order BBB which was working fine with Chrome and Restarted the BBB but the issue still exists on BBB 2.5.5

I see this issue appears on Chrome on Windows and Android.

is there any fast fix on this?
have you any experience with this issue?

Thanks

**Comment by amg-web** [comment]
@alizamanir 
try clean browser's cache. no issues for me

**Comment by alizamanir** [comment]
Thanks for your reply

Its not the cache problem. I tested it on different systems and different clients reported this issue to us.
the playback on Chrome has issue and its not possible to move the play back forward or backward.

**Comment by alizamanir** [comment]
I moved the following folder:

/var/bigbluebutton/playback/presentation/2.3

and reinstalled the BBB using:

https://github.com/bigbluebutton/bbb-install

and it re-installed the BBB but the 2.3 folder in the /var/bigbluebutton/playback/presentation was not created.

then I installed the play-back using the following link:

https://github.com/bigbluebutton/bbb-playback

then the 2.3 folder created but the issue still exists and its not possible to move forward or backward the presentation.

is there any way to switch back to 2.5.4 instead of the 2.5.5?


Thanks
